lib/resorts/falls-creek/resort.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Falls Creek",
+  "url": {
+    "host": "https://www.fallscreek.com.au",
+    "pathname": "/snowreport/"
+  },
+  "tags": [
+    "Australia",
+    "Victoria"
+  ],
+  "ll": [
+    147.287,
+    36.8659
```

Comment:
unless you guys moved Australia overnight latitude should be a negative number ;-)